docs/source/conf.py

```diff
@@ -172,6 +172,7 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
+    "header_links_before_dropdown": 4,
```

Comment:
Yea, I just pushed a change, setting `header_links_before_dropdown` to 5 so `Changelog` will show and any other entries will go under the `More` in case anything gets added down the line. We could also get rid of that line altogether if that's preferred. 

![nav2](https://github.com/user-attachments/assets/59b2878e-af3c-4613-b899-fcf10b200fa4)